### PR TITLE
Fsh/images

### PR DESCRIPTION
**Pull request by fharding1**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -255,3 +255,13 @@ Sends the report schema.
   "movedBuckets": "number"
 }
 ```
+
+---
+
+## /photo/{team}
+
+Responses with the (binary) photo for given team.
```
Comment (BrendanBurkhart) [inline docs/api.md]: Did you mean "Responds with"?

File: internal/server/server.go [hunk 2]
```diff
@@ -140,56 +135,77 @@ func (s *Server) newRouter() *mux.Router {
...
 	router.Handle("/analysis/{eventKey}/{matchKey}/{color}", cors(http.HandlerFunc(s.allianceAnalysisHandler), []string{"GET"}))
 
 	return router
 }
 
-func cors(next http.Handler, allowedMethods []string) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Access-Control-Allow-Origin", "*")
-		w.Header().Set("Access-Control-Allow-Methods", strings.Join(append(allowedMethods, "OPTIONS"), ","))
+const imgurFormat = "http://i.imgur.com/%sl.jpg"
```
Comment (BrendanBurkhart) [inline internal/server/server.go]: Ok I don't really know anything about Golang constants, would it be better to stick this in utils or would it just be confusing to have it there?

File: internal/server/server.go [hunk 3]
```diff
@@ -140,56 +135,77 @@ func (s *Server) newRouter() *mux.Router {
...
+		if err != nil {
+			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+			s.logger.LogRequestError(r, fmt.Errorf("getting team media: %v", err))
 			return
 		}
 
-		next.ServeHTTP(w, r)
-	})
-}
+		for _, m := range media {
```
Comment (BrendanBurkhart) [inline internal/server/server.go]: Shouldn't this break once it has found a url?

File: docs/api.md (hunk 1 above)
Comment (fharding1) [reply docs/api.md]: No /s 😂

Comment (fharding1) [reply docs/api.md]: totally on purpose

File: internal/server/server.go (hunk 2 above)
Comment (fharding1) [reply internal/server/server.go]: It is used in the function below, it makes sense to have it right above.

**Comment by fharding1** [comment]
🎉 indeed

**Pull request merged**